### DISCUSSION TITLE
User survey

### DISCUSSION THREAD

**Discussion by jbtronics** [Part-DB/Part-DB-server#231] [Announcements]
# User survey

**If you are using Part-DB it would be helpful if you fill out this short survey on your usage of Part-DB (Google Forms): https://forms.gle/RyDb2DY11H9hpfm6A .**

The information if you could upgrade your PHP version to PHP 8.1 (or have already done that), as many new versions of PHP libraries now only support PHP 8.1 and also new versions of Symfony (the framework Part-DB) require PHP 8.1. The current version Symfony 5.4 will receive bugfixes until November 2024, but it would be good to be able to upgrade some time before. Also the new features of PHP 8.1 would improve code quality of Part-DB.

**Comment by Pyromane** [comment]
Thank you for the further development!
I filled out the survey.
